modules/apps/collaboration/message-boards/message-boards-parser-bbcode/src/main/java/com/liferay/message/boards/parser/bbcode/internal/HtmlBBCodeTranslatorImpl.java

```diff
@@ -334,7 +334,7 @@ protected void handleData(
 		StringBundler sb, List<BBCodeItem> bbCodeItems, Stack<String> tags,
 		IntegerWrapper marker, BBCodeItem bbCodeItem) {
 
-		String value = HtmlUtil.escape(bbCodeItem.getValue());
```

Comment:
where is the value be printed ?

Reply:
This value will be printed after clicking the "Subscribe to recent posts" link in the "Message Boards" section.

![escapeValue](https://user-images.githubusercontent.com/73691202/113826271-28d2ba00-97ac-11eb-8445-2d76137983d5.png)

Reply:
Can you take a screen shot before your fix ?

Reply:
This issue occurs in the XML file. I have raised this on Zendesk and Mr. Minh Chau Dang commented this is required to escape. So I don't need to unescape this case. I fixed my code.
Thank you so much and if you need more information please take a look at this ticket: https://liferay-support.zendesk.com/agent/tickets/46880